modules/lightning_features/lightning_media/src/BundleResolverBase.php

```diff
@@ -13,6 +13,8 @@
  */
 abstract class BundleResolverBase extends PluginBase implements BundleResolverInterface, ContainerFactoryPluginInterface {
 
+  use SourceFieldTrait;
```

Comment:
I'm not sure about SourceFieldTrait being part of BundleResolverBase. It probably won't harm anything, but it isn't architecturally "pure", since the bundle resolver API shouldn't make assumptions about the media bundles in question.